Jenkinsfile

```diff
@@ -22,6 +22,7 @@ pipeline {
                 dir("server"){
                     sh "$INMANTA_TEST_ENV/bin/python3 -m tox --recreate"
                 }
+                sh 'rm -rf node_modules; npm i --also=dev; xvfb-run npm run test'
```

Comment:
Perhaps separate client and server tests in different stages?

Reply:
Good idea, I will do that

Reply:
Did it in https://github.com/inmanta/vscode-inmanta/pull/33/commits/decabc3b9573bc0da5f374725efb20e2621c1a7b